CONTRIBUTING.md

```diff
@@ -38,7 +38,7 @@ Before submitting a pull request:
 
 ## Run Tests
 
-The full test suite requires PHP cli with mongodb extension, a running MongoDB server and a running MySQL server.
+The full test suite requires PHP cli with mongodb extension, a running MongoDB server.

```

Comment:
Is MySQL still required for testing hybrid relationships? If not, you can drop the second sentence, but the suggestion also improves the wording of the first part.

```suggestion
The full test suite requires PHP cli with mongodb extension and a running MongoDB server. A MySQL server is required to test hybrid relationships.
```

Reply:
MySQL is not necessary since I switched to sqlite: https://github.com/mongodb/laravel-mongodb/pull/2616

Reply:
I added a note for testing transactions.